src/statmodels.jl

```diff
@@ -542,6 +542,56 @@ function show(io::IO, ct::CoefTable)
     nothing
 end
 
+show(mime::MIME"text/markdown", ct::CoefTable) = show(Base.stdout, mime, ct)
```

Comment:
Why add this method? AFAICT this form isn't documented in `?show`, and it's not tested either.

Reply:
Convenience, but I'm not attached to it, so happy to delete.

Reply:
`methods(show)` doesn't include any similar method, so I'd rather remove it.